testing-toolkit/collections/payer-tests/outbound_transfer.json

```diff
@@ -1,195 +0,0 @@
-{
```

Comment:
Why this file is deleted?

Reply:
This file is not deleted , just renamed to the outbound_transfer_fx.json , with an added test case to it

Reply:
I think we should have this file intact and have another file for fx usecase.

Reply:
Normal transfer and fx transfers are different and we should have both files.

Reply:
Okay let me separate the transfers in different files then

Reply:
Done